nestedtensor/nested/nested.py

```diff
@@ -213,11 +213,63 @@ def nested_stride(self, dim=None):
     def __torch_function__(self, func, types, args=(), kwargs=None):
         _local_func = None
         if func in NestedTensor.__C_functions:
+            impl_args = [a._impl if isinstance(a, NestedTensor) else a for a in args]
```

Comment:
None of these changes should be required. You shouldn't have to change anything in nested.py.